src/templates/fundraising/index.js

```diff
@@ -19,92 +11,9 @@ export default {
     Initialize a transparent and accountable crowdfunding campaign for your
     organization.
   `,
-  // longdesc: ``,
-  // caseStudyUrl: 'https://aragon.org/case-study/fundraising',
-  // TODO: Insert proper user guide URL
   userGuide: 'https://help.aragon.org/',
-  // sourceCodeUrl: 'https://github.com/aragon/dao-templates/tree/master/templates/',
   registry: 'aragonpm.eth',
   modules: [],
-  screens: [
-    [data => completeDomain(data.domain) || 'Claim domain', ClaimDomain],
-    ['Configure template', Voting],
-    ['Configure template', Tokens],
-    [
-      'Review information',
-      ({ back, data, next }) => (
-        <Review
-          back={back}
-          data={data}
-          next={next}
-          items={[
-            {
-              label: 'General info',
-              fields: [
-                ['Organization template', 'Fundraising'],
-                ['Name', completeDomain(data.domain)],
-              ],
-            },
-            {
-              label: 'Voting app',
-              fields: Voting.formatReviewFields(data.voting),
-            },
-            {
-              label: 'Tokens app',
-              fields: Tokens.formatReviewFields(data.tokens),
-            },
-          ]}
-        />
-      ),
-    ],
-  ],
-  prepareTransactions(createTx, data) {
-    const hasPayroll = false
-
-    const {
-      tokenName,
-      tokenSymbol,
-      subdomain,
-      holders,
-      stakes,
-      votingSettings,
-      financePeriod,
-    } = data
-
-    if (!hasPayroll) {
-      return [
-        {
-          name: 'Create organization',
-          transaction: createTx('newTokenAndInstance', [
-            tokenName,
-            tokenSymbol,
-            subdomain,
-            holders,
-            stakes,
-            votingSettings,
-            financePeriod,
-            true,
-          ]),
-        },
-      ]
-    }
-
-    return [
-      {
-        name: 'Create token',
-        transaction: createTx('newToken', [tokenName, tokenSymbol]),
-      },
-      {
-        name: 'Create organization',
-        transaction: createTx('newInstance', [
-          subdomain,
-          holders,
-          stakes,
-          votingSettings,
-          financePeriod,
-          true,
-        ]),
-      },
-    ]
-  },
+  screens: [],
+  prepareTransactions: () => [],
```

Comment:
Could this be set via a default value?

Reply:
Yes! Actually we don’t even need it: https://github.com/aragon/aragon/pull/1058/commits/48939b2b9f1a6e9c80d6d46edac48b8cc9c6b7af